test/integration/testpaths/testpath_vmlc.py

```diff
@@ -176,6 +176,19 @@ def setUpClass(cls):
         cls.zone = get_zone(cls.apiclient)
         cls._cleanup = []
 
+        cls.skiptest = False
```

Comment:
Does this test require any special storage pool? or with any tags? If not, there is no reason to add this check as it is expected that storage pool is present in setup.